src/h5web/vis-packs/nexus/visualizations.ts

```diff
@@ -35,11 +47,11 @@ export const NEXUS_VIS: Record<NexusVis, VisDef> = {
     ConfigProvider: HeatmapConfigProvider,
   },
 
-  [NexusVis.NxComplex]: {
+  [NexusVis.NxComplexImage]: {
     name: NexusVis.NxImage,
     Icon: FiMap,
     Toolbar: ComplexToolbar,
-    Container: NxComplexContainer,
+    Container: NxComplexImageContainer,
```

Comment:
Renamed `NxComplex` into `NxComplexImage` in accordance with `NxComplexSpectrum`